zanata-war/src/main/webapp/WEB-INF/layout/version-group/projects-tab.xhtml

```diff
@@ -162,9 +161,9 @@
       <zanata:loader jsHandle="true" layout="inline" type="loader--small" visible="false"
         id="projectsTab-project-label-loader"/>
 
-      <s:span id="project-label" styleClass="i__text--right">
+      <h:panelGroup id="project-label" class="i__text--right">
```

Comment:
Good catch. Fixed all of them

On Wed, Sep 23, 2015 at 2:24 PM, Alex Eng <notifications@github.com> wrote:

> In
> zanata-war/src/main/webapp/WEB-INF/layout/version-group/projects-tab.xhtml
> <https://github.com/zanata/zanata-server/pull/928#discussion_r40170157>:
>
> > @@ -162,9 +161,9 @@
> >        <zanata:loader jsHandle="true" layout="inline" type="loader--small" visible="false"
> >          id="projectsTab-project-label-loader"/>
> >
> > -      <s:span id="project-label" styleClass="i__text--right">
> > +      <h:panelGroup id="project-label" class="i__text--right">
>
> styleClass?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/928/files#r40170157>.
>



-- 
Patrick Huang
Senior Software Engineer
Engineering - Internationalisation
Red Hat, Asia-Pacific Pty Ltd
Level 1, 193 North Quay
Brisbane 4000
Office: +61 7 3514 8278
Fax: +61 7 3514 8199
IRC: pahuang
github: github.com/huangp
Website: www.redhat.com